include/rand.h

```diff
@@ -1,18 +1,10 @@
 #ifndef RAND_H
 #define RAND_H
 
-#include "ultra64.h"
+// For convenience
+#include "libc64/qrand.h"
 
-u32 Rand_Next(void);
-void Rand_Seed(u32 seed);
-f32 Rand_ZeroOne(void);
-void Rand_Seed_Variable(u32* rndNum, u32 seed);
-u32 Rand_Next_Variable(u32* rndNum);
-f32 Rand_ZeroOne_Variable(u32* rndNum);
```

Comment:
think we should also explore the idea of original names + wrappers in our own game side header like I mentioned in the other PR. doesnt have to happen here tho